Rakefile

```diff
@@ -17,7 +17,7 @@ namespace :style do
   desc 'Run Chef style checks'
   task :chef do
     sh '/opt/chefdk/embedded/bin/foodcritic --version'
-    sh '/opt/chefdk/embedded/bin/foodcritic -f any . --exclude spec -t ~FC059'
+    sh '/opt/chefdk/embedded/bin/foodcritic -f any . --exclude spec -t ~FC059 -t ~FC017'
```

Comment:
Just a heads up we probably want to just make a .foodcritic file with these in it